components/frontend/src/changelog/ChangeLog.test.jsx

```diff
@@ -72,13 +70,13 @@ it("loads more changes", async () => {
 })
 
 it("shows error when loading more changes fails", async () => {
-    changelogApi.getChangelog.mockImplementation(() => Promise.resolve({ changelog: [] }))
     const { container } = await renderChangeLog({ sourceUuid: "uuid" })
-    changelogApi.getChangelog.mockImplementation(() => Promise.reject(new Error("Couldn't retrieve changelog")))
+    fetchServerApi.fetchServerApi.mockImplementation(() => Promise.reject(new Error("Couldn't retrieve changelog")))
+    const showMessage = vi.spyOn(toast, "showMessage")
     await act(async () => fireEvent.click(screen.getByText(/Load more changes/)))
     await waitFor(async () => {
-        expect(toast.showMessage).toHaveBeenCalledTimes(1)
+        expect(showMessage).toHaveBeenCalledTimes(1)
         await expectNoAccessibilityViolations(container)
     })
-    expectNrEventsToBe(0)
+    expectNrEventsToBe(1)
```

Comment:
Why does this change now mean there was an event?

Reply:
Because the default `mockImplementation` was changed. Before, it returned no changes, now it returns 1.